examples/TemplateGenerationIterative.ipynb

```diff
@@ -303,73 +351,63 @@
      ]
     }
    ],
    "source": [
-    "meshes = list()\n",
+    "dense_meshes = generate_meshes(dense_downsampled_images)\n",
     "\n",
-    "for image in downsampled_images:\n",
-    "    mesh_source = MeshSourceType.New()\n",
-    "    mesh_source.SetObjectValue(FEMUR_OBJECT_PIXEL_VALUE)\n",
-    "    mesh_source.SetInput(image)\n",
-    "    mesh_source.Update()\n",
-    "    \n",
-    "    mesh_output = mesh_source.GetOutput()\n",
-    "    meshes.append(mesh_output)\n",
-    "    \n",
-    "print('Average mesh points: ' +\n",
-    "      str(sum([mesh.GetNumberOfPoints() for mesh in meshes]) / len(meshes)))"
+    "# Expect ~200K vertices\n",
+    "print('Average dense mesh points: ' +\n",
+    "      str(sum([mesh.GetNumberOfPoints() for mesh in dense_meshes]) / len(dense_meshes)))"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 18,
    "metadata": {},
-   "outputs": [],
-   "source": [
-    "# Write out each mesh to disk\n",
-    "for i in range(0,len(meshes)):\n",
-    "    itk.meshwrite(meshes[i], f'{MESH_OUTPUT_FOLDER}{MESH_FILENAMES[i]}')"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 19,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "# TODO visualize with itkwidgets\n",
-    "# view(geometries=meshes)"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Average sparse mesh points: 4373.357142857143\n"
+     ]
+    }
+   ],
    "source": [
-    "### Select Template Mesh\n",
-    "It is now necessary to select a mesh that will act as the 'standard' for correspondence point updates from this point forward. The first mesh in the list is arbitrarily selected as this standard template."
+    "sparse_meshes = generate_meshes(sparse_downsampled_images)\n",
+    "\n",
+    "# Expect ~5K vertices\n",
+    "print('Average sparse mesh points: ' +\n",
+    "      str(sum([mesh.GetNumberOfPoints() for mesh in sparse_meshes]) / len(sparse_meshes)))"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
-   "metadata": {},
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
    "outputs": [],
    "source": [
-    "TEMPLATE_MESH_INDEX = 0\n",
-    "template_mesh = meshes[TEMPLATE_MESH_INDEX]"
+    "# Write out each mesh to disk\n",
+    "for i in range(0,len(dense_meshes)):\n",
+    "    itk.meshwrite(dense_meshes[i], f'{DENSE_MESH_OUTPUT_FOLDER}{MESH_FILENAMES[i]}')\n",
+    "\n",
+    "for i in range(0,len(sparse_meshes)):\n",
+    "    itk.meshwrite(sparse_meshes[i], f'{TEMPLATE_OUTPUT_FOLDER}{MESH_FILENAMES[i]}')"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 21,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# FIXME visualize template\n",
-    "# view(geometries=[template_mesh])"
+    "# visualize with itkwidgets\n",
+    "#view(geometries=meshes)"
```

Comment:
Either fix or remove?

Reply:
meshes -> sparse_meshes or dense_meshes

Reply:
Fixed `meshes` -> `sparse_meshes`.

`view(geometries=[])` currently seems to be broken due to ITK/ITKMeshToPolyData/ITKWidgets version compatibility issues. These cells should be operable pending ITKWidgets update via @thewtex .